Modern/ops/src/main/java/org/bonej/ops/ellipsoid/EllipsoidOptimisationStrategy.java

```diff
@@ -537,7 +537,7 @@ public QuickEllipsoid calculate(byte[][] pixels, Vector3d seedPoint) {
 
 		final long stop = System.currentTimeMillis();
 
-		logService.debug("Optimised ellipsoid in " + (stop - start) + " ms after " + totalIterations + " iterations ("
+		logService.info("Optimised ellipsoid in " + (stop - start) + " ms after " + totalIterations + " iterations ("
```

Comment:
Can you also use the status bar to show _n_ out of _total_ ellipsoids have been fitted?

Reply:
For me, the problem is how to get _n_ out of a parallelStream() - that is the bit I don't understand/know if it is possible?

Reply:
>  how to get _n_ out of a parallelStream()

No idea, no way I have found.

Reply:
You could do
```
final AtomigInteger progress = new AtomicInteger();
...
stream.parallel().peek(x -> {
  statusService.showProgress(progress.getAndIncrement(), max);
}).forEach(x -> ...);
'''

Reply:
Ooo interesting - will test!